tests/unit/test_charm.py

```diff
@@ -1,10 +1,22 @@
+import inspect
+from pathlib import Path
 from unittest import TestCase, mock
 
+import yaml
 from ops.testing import Harness
 
 import charm
 
 
+def charm_config() -> str:
+    """Return the charm configuration as a string readfrom charmcraft.yaml."""
+    filename = inspect.getfile(charm.UbuntuCharm)
+    charm_dir = Path(filename).parents[1]
+    content = (charm_dir / "charmcraft.yaml").read_text()
+    config = yaml.safe_load(content)["config"]
+    return yaml.safe_dump(config)
+
+
```

Comment:
`ops>2` doesn't require this -- but in order to maintain bionic, we can't use `ops>2`